### PR TITLE
INDI: Implement telescope syncing

### PR DESCRIPTION
**Pull request by akosiaris**
### Description
Up to now, telescope sync when INDI was used wasn't implemented. Add
support for it. Add a new syncPosition() procedure that will first set
the SYNC ON_COORD_SET switch member and then send the coordinates. Make
sure to unset the switch member at the end.

While at it, make sure that setPosition() procedure is also setting the
TRACK member of the ON_COORD_SET switch to avoid undesired state before
sending the coordinates

Implement the corresponding client function that was a noop up to now

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

### How Has This Been Tested?

**Test Configuration**: flatpak build based on the official flatpak manifest. 
* Operating system: Debian buster, already running Stellarium using the flatpak build.
* Graphics Card: Intel Iris Graphics 5100. Whatever driver version currently ships with Debian Buster.

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by akosiaris** [comment]
Greetings fails with:
```
 Adding message: Thanks for adding your first pull request to Stellarium. If you have questions, please do not hesitate to contact us. to pull request 1048
##[error]Resource not accessible by integration
```

I have no idea why though.

**Comment by alex-w** [comment]
Are you checked it on real hardware?

**Comment by akosiaris** [comment]
> Are you checked it on real hardware?

No, I haven't found the time yet. I 'll do so once the clouds are gone.

File: plugins/TelescopeControl/src/INDI/TelescopeClientINDI.cpp [hunk 1]
```diff
@@ -94,8 +94,26 @@ void TelescopeClientINDI::telescopeGoto(const Vec3d &positionJ2000, StelObjectP
...
+	double latitudeRad = posJ2000[1] * 180.0 / M_PI;
+
+
+	INDIConnection::Coordinates positionJNow = mConnection.position();
+	positionJNow.RA = longitudeRad;
+	positionJNow.DEC = latitudeRad;
+
+	// unpark telescope before slewing!
+	// TODO: Add commands and buttons for park/unpark telescope for all telescopes
+	mConnection.unParkTelescope();
```
Comment (d33psky) [inline plugins/TelescopeControl/src/INDI/TelescopeClientINDI.cpp:115]: I do not think this is the place to call *this* particular command.
It's fine to 'make sure' the mount is unparked before giving it a sync command in another way, but we should not unpark at every sync.

Comment (alex-w) [reply plugins/TelescopeControl/src/INDI/TelescopeClientINDI.cpp:115]: The mount will not unpark here if she is unparked already. But possible we should add 2 buttons (Park, Unpark) for all telescopes to allow better management of mounts.

**Comment by akosiaris** [comment]
Managed to find the weather and time to test this. Unfortunately it turns out that the traffic flowing to indiserver from stellarium did not correctly set the SYNC member of ON_COORD_SET switch. I 've submitted #1065 as a tested fix.

**Pull request merged**